pages/index.tsx

```diff
@@ -1,21 +1,37 @@
 import { Box } from '@chakra-ui/react'
 import { useEffect } from 'react'
 import type { NextPage } from 'next'
-import Head from 'next/head'
 import Hero from '../components/Hero'
 import * as FullStory from '@fullstory/browser'
+import { NextSeo } from 'next-seo'
 
 const Home: NextPage = () => {
   useEffect(() => {
     FullStory.init({ orgId: 'o-1CKVPB-na1' })
   }, [])
   return (
     <div>
-      <Head>
-        <title>Developer DAO School of Code</title>
-        <meta name="description" content="Developer DAO's school of code" />
-        <link rel="icon" href="/favicon.ico" />
-      </Head>
+      <NextSeo
+        title="Developer DAO School of Code"
+        description="Developer DAO's school of code"
```

Comment:
For description, how about "School of Code is an open-source education platform created by the Developer DAO."